loader/extends.go

```diff
@@ -68,7 +68,10 @@ func applyServiceExtends(ctx context.Context, name string, services map[string]a
 	)
 	switch v := extends.(type) {
 	case map[string]any:
-		ref = v["service"].(string)
+		ref, ok = v["service"].(string)
+		if !ok {
+			return nil, fmt.Errorf("extends.%s.service is required", name)
```

Comment:
json schema already declares `"required": ["service"]`, so why do we end up seing this?